README.md

```diff
@@ -20,33 +20,51 @@ Consider using [fluo-dev] to run these requirements
 
 ### Configure your environment
 
-First, you must create the configuration files `data.yml` and `dropwizard.yml` files in the `conf/` 
-directory and edit them for your environment:
+First, you must create the configuration file `data.yml` in the `conf/` directory and edit it
+for your environment:
 
     cp conf/data.yml.example conf/data.yml
-    cp conf/dropwizard.yml.example conf/dropwizard.yml
+
+There are a few environment variables that need to be set to run these scripts (see `conf/webindex-env.sh.example`
+for a list).  If you don't want to set them in your `~/.bashrc`, create `webindex-env.sh` in `conf/` and set them.
+
+    cp conf/webindex-env.sh.example conf/webindex-env.sh
 
 ### Copy CommonCrawl data from AWS into HDFS
 
 CommonCrawl data sets are hosted in S3.  The following command will run a Spark job that copies
-data files from AWS into HDFS.  This script is configured by `conf/data.yml` where you can specify
-the HDFS data directory, number of files to copy, and number of executors to run.
+data files from AWS into HDFS `init/` and `load/` directoires.  This script is configured by 
+`conf/data.yml` where you can specify the number of files to copy into each directory:
 
-    ./bin/copy.sh
+    ./bin/webindex copy
```

Comment:
Something is missing from the instructions here.  This did not work when I first ran it.  I had to do `webindex copy --build`

```
[ERROR] Failed to execute goal on project webindex-data: Could not resolve dependencies for project io.fluo:webindex-data:jar:0.0.1-SNAPSHOT: Could not find artifact io.fluo:webindex-core:jar:0.0.1-SNAPSHOT in sonatype-nexus-snapshots (https://oss.sonatype.org/content/repositories/snapshots) -> [Help 1]
```